test/filter.js

```diff
@@ -9,6 +9,13 @@ test('With a where option', t => {
   run('trees', options, 12105, t)
 })
 
+test('With a where option with multiple statements', t => {
+  const options = {
```

Comment:
@dmfenton take a look now. Simply changed the Street_Name statement to check for equality